test/integration/package_int_test.go

```diff
@@ -348,26 +348,29 @@ func (suite *PackageIntegrationTestSuite) TestPackage_operation() {
 
 	suite.Run("install", func() {
 		cp := ts.Spawn("install", "dateparser@0.7.2")
-		cp.ExpectRe("(?:Package added|project is currently building)")
+		cp.ExpectRe("(?:Package added|project is currently building)", 60*time.Second)
```

Comment:
Maybe we can use a package with less dependencies here?  Maybe `JSON` and `JSON@4.02`?

Reply:
Good idea. I opted for `urllib3` as this is a python project